docs/onboarding_to_cluster.md

```diff
@@ -97,14 +92,23 @@ users:
   - USER_2
 ```
 
-Now please encrypt the file with sops and verify it contains a fingerprint of both GPG keys: the OperateFirst master key and your team's key.
+> Note that the USER_n value is the email used to login via SSO in the preceeding steps.
+
+Encrypt the file with sops, you can find the key to import from [here](https://github.com/operate-first/apps/tree/master/cluster-scope/overlays/moc#secret-management):
+
+```sh
+$ sops --encrypt --encrypted-regex="^users$" --pgp="0508677DD04952D06A943D5B4DC4116D360E3276" groups/GROUP_NAME.yaml > groups/GROUP_NAME.enc.yaml
+$ grep "fp: " groups/GROUP_NAME.enc.yaml
+        fp: 0508677DD04952D06A943D5B4DC4116D360E3276
+```
+
+If you or your team would like the ability to decrypt this file then you will need to add a GPG key fingerprint for which you or your team has decrypt privileges to:
```

Comment:
I am not sure that the phrase "a GPG key fingerprint for which you or your team has decrypt privileges" make sense (generally, either you have the private key or you don't -- there aren't "privileges").  Maybe:

> If you or your team needs the ability to decrypt this file, you may include additional PGP key fingerprints in the `sops` command:

Reply:
updated